### PR TITLE
add intro help text

### PR DESCRIPTION
**Pull request by lb-**
- see https://www.guidodiepen.nl/2021/05/include-help-instructions-in-cookiecutter-prompts/
- this will help the cookiecutter user know that the project name does not need to have 'Wagtail' in it
- moving the names/usernames to after the project name definition

### CODE REVIEW

File: cookiecutter.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
 {
-    "full_name": "Karl Hobley",
-    "email": "karl@kaed.uk",
-    "github_username": "kaedroho",
+    " ": "]\nCookiecutter Wagtail Package - Use a project name without `Wagtail` as the prefix, this wil be added for you.\n[Please press enter to continue",
```
Comment (kaedroho) [inline cookiecutter.json:2]: Typo: wil

**Review by kaedroho — CHANGES_REQUESTED** [review]
Minor typo, but otherwise looks good to me!

File: cookiecutter.json (hunk 1 above)
Comment (lb-) [reply cookiecutter.json]: thanks for that, fixed up & rebased now.

**Comment by lb-** [comment]
Thanks!

**Pull request merged**